docs/using-airbyte/mappings.md

```diff
@@ -64,44 +66,44 @@ There are many reasons you might want to filter rows.
 - **Optimization**: Smaller datasets require less processing and storage, and you can query them faster.
 - **Compliance**: You don't want to keep data longer than is needed for a defined business purpose, and you want to ensure you don't accidentally archive data from individuals who opted out of data collection.
 
+:::note
+You can only filter fields whose type is string and number.
+:::
+
+## Considerations and limitations
+
+Before you begin mapping fields, consider the following.
+
+- If you set up mapping on an existing connection, Airbyte prompts you to clear your stream and the data in your destination. You almost certainly should do this. Not doing this could result in undesirable or unexpected behavior. For example, if you set up hashing on a field that contains personally identifiable information (PII), you want all data in that column to be hashed, not just new data. Clearing your destination accomplishes this. However, if you're dealing with an extremely large database, a complete resync might take a significant amount of time.
```

Comment:
For most destinations, we will recommend a `refresh` from within the UI, not a clear.  The only exception to this is if the destination does not support refreshes (only a few Marketplace destinations are this way).  

However, for this operation you are right that users will probably want a `clear` unless their sync mode uses `Full Refresh`.